catalogue_pipeline/transformer/src/main/scala/uk/ac/wellcome/transformer/transformers/miro/MiroGenres.scala

```diff
@@ -1,17 +1,23 @@
 package uk.ac.wellcome.transformer.transformers.miro
 
-import uk.ac.wellcome.models.work.internal.{AbstractConcept, Concept, Genre}
+import uk.ac.wellcome.models.work.internal.{
+  AbstractConcept,
+  Concept,
+  Genre,
+  MaybeDisplayable,
+  Unidentifiable
+}
 import uk.ac.wellcome.transformer.source.MiroTransformableData
 
 trait MiroGenres {
   def getGenres(
-    miroData: MiroTransformableData): List[Genre[AbstractConcept]] = {
+    miroData: MiroTransformableData): List[Genre[MaybeDisplayable[AbstractConcept]]] = {
     // Populate the genres field.  This is based on two fields in the XML,
     // <image_phys_format> and <image_lc_genre>.
     (miroData.physFormat.toList ++ miroData.lcGenre.toList).map { label =>
-      Genre[AbstractConcept](
+      Genre[MaybeDisplayable[AbstractConcept]](
```

Comment:
Since it's never going to be anything other than `Unidentifiable` for miro, can we not type it as `Genre[Unidentifiable[AbstractConcept]]` here and above?

Reply:
Actually it could probably be `Genre[Unidentifiable[Concept]]` cause it's never going to be anything other than `Concept` either